projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -218,3 +218,103 @@ export class MovieModule {}
 **Note:** Running an effects class multiple times, either by `forRoot()` or `forFeature()`, (for example via different lazy loaded modules) will not cause Effects to run multiple times. There is no functional difference between effects loaded by `forRoot()` and `forFeature()`; the important difference between the functions is that `forRoot()` sets up the providers required for effects.
 
 </div>
+
+## Incorporating State
+
+If additional metadata is needed to perform an effect besides the initiating action's `type`, we should rely on passed metadata from an action creator's `props` method.
+
+Let's look at an example of an action initiating a login request using an effect with additional passed metadata:
+
+<code-example header="login-page.actions.ts">
+import { createAction, props } from '@ngrx/store';
+import { Credentials } from '@example-app/auth/models/user';
```

Comment:
Change the `@example-app` to relative imports. Maybe some day we'll be able import snippets from the example app directly 😃

Reply:
switched all the `@example-app` imports to relative imports in 5c21180.